server/views/pages/reviewPlan/exemption/index.njk

```diff
@@ -0,0 +1,207 @@
+{% extends "../../../partials/layout.njk" %}
+
+{% set pageTitle = "Exemption" %}
+{% set pageId = "exemption" %}
```

Comment:
I think the pageId and pageTitle need to be a bit more explicit. In our other pages we have this type of thing:

```
{% set pageTitle = "Who completed the prisoner's plan review?" %}

{% set pageId = "review-plan-who-completed-review" %}
```

So here perhaps the pageId  should be `review-plan-exemption-reason`; and the pageTitle should be something like `Put the prisoner's review on hold` or something? 🤷‍♂️  (we're not allowed to put the prisoner name/id in the `pageTitle` here, so we can't just copy/duplicate the H1)

Also (thinking about the other pages we are about to write for this feature) I think the folder would be better off being called `exemptionReason` (because the other pages might be in folders `exemptionConfirm` (or `exemptionAreYouSure`), and `exemptionComplete`)